.stylelintrc

```diff
@@ -1,3 +1,13 @@
 {
-  "extends": "stylelint-config-standard"
+	"extends": "stylelint-config-standard",
+	"rules": {
+		"property-no-unknown": [
+			true,
+			{
+				"ignoreProperties": [
+					"font-size-responsive"
```

Comment:
I would add all the possible options in here, so that if someone makes use of them they're not facing down unclear linting errors. Here's the list I'm using on a current project:

```
"font-range",
        "min-font-size",
        "max-font-size",
        "lower-font-range",
        "upper-font-range",
        "line-height-range",
        "min-line-height",
        "max-line-height",
        "lower-line-height-range",
        "upper-line-height-range",
        "letter-spacing-range",
        "min-letter-spacing",
        "max-letter-spacing",
        "lower-letter-spacing-range",
        "upper-letter-spacing-range"
```

Reply:
Hey Dan,

I think I might be using a different PostCSS plugin because I don't see these properties in [the plugin repo](https://github.com/ccurtin/postcss-responsive-font) (checked the codebase and docs).

If you have another one that you prefer, link it here and I can switch over to that one (and add the properties listed above).